tutorials/tutorial11_inflows.ipynb

```diff
@@ -348,7 +334,7 @@
     "- `depart_speed`: this parameter lets you specify the speed at which the vehicles will enter the network. It should be a positive `float`, in meters per second. If this speed is unsafe, the departure of the vehicles is delayed. Just like for `depart_lane`, there are other options for this parameter, which are the following strings:\n",
     "\n",
     "    - `\"random\"`: vehicles enter the edge with a random speed between 0 and the speed limit on the edge. The entering speed may be adapted to ensure that a safe distance to the leading vehicle is kept\n",
-    "    - `\"speedLimit\"`: vehicles enter the edge with the maximum speed that is allowed on this edge. If that speed is unsafe, the departure is delayed.\n",
```

Comment:
don't delete this. replace "speedLimit" with "max", see: https://sumo.dlr.de/docs/Simulation/VehicleInsertion.html